.github/workflows/consistency-functional.yml

```diff
@@ -119,16 +121,17 @@ jobs:
           ADMIN_PASSWORD=devstack
           SERVICE_TOKEN=devstack
           DATABASE_PASSWORD=root
+          DATABASE_TYPE=mysql
           HOST_IP=127.0.0.1
           SERVICE_HOST=127.0.0.1
           MYSQL_HOST=127.0.0.1
           RABBIT_HOST=127.0.0.1
           RABBIT_USERID=guest
           RABBIT_PASSWORD=guest
           # Pre-requisite
-          ENABLED_SERVICES=mysql,key
+          ENABLED_SERVICES=key
           # Neutron
-          ENABLED_SERVICES+=,q-svc,q-agt,q-dhcp,q-l3,q-meta,neutron
```

Comment:
Removing neutron services it finished the install we need to debug why is not working, maybe something related to the DB

Reply:
It still looks like we have Neutron API available. I guess that's the `q-svc` in the list. Not sure what `neutron` stands for but the error with connecting to port 6640 that you pasted earlier is related to OVS, so perhaps OVS is the culprit here. Let's merge to unblock CI.